sample-courgette-mobile-conf.js

```diff
@@ -0,0 +1,140 @@
+const path = require('path');
+const specsPath = 'ui-tests';
+const outputPath = 'uiTestResult';
+const courgettePath = 'node_modules/courgette/uiTestHelpers';
+
+const platform = process.env.PLATFORM;
+
+const ANDROID_APP_PATH = './android/app/build/outputs/apk/debug/app-debug.apk';
+const IOS_APP_PATH = './ios/build/Products/Release-iphonesimulator/APPNAME.app';
+
+const ALL_CAPABILITIES = {
+  android: [
+    {
+      platformName: 'android',
+      platformVersion: '10',
+      deviceName: 'Android Emulator',
+      app: ANDROID_APP_PATH,
+    },
+  ],
+  ios: [
+    {
+      platformName: 'ios',
+      platformVersion: '13.3',
+      deviceName: 'iPhone 11',
+      automationName: 'XCUITest',
+      app: IOS_APP_PATH,
+      // showXcodeLog: true,
```

Comment:
what's this doing here?

Reply:
There's a lot of commented out code, do you need it? :)

Reply:
Sorry, forgot to mark it as a WIP. Still need to finish these bits off